src/js/subtraction/components/__tests__/Subtraction.test.tsx

```diff
@@ -0,0 +1,30 @@
+import { screen, waitFor } from "@testing-library/react";
+import { createBrowserHistory } from "history";
+import nock from "nock";
+import React from "react";
+import { beforeEach, describe, expect, it } from "vitest";
+import { renderWithRouter } from "../../../../tests/setupTests";
+import Subtraction from "../Subtraction";
+
+const subtractionList = {
+    documents: [{ id: "foo", name: "bar", user: { handle: "test" } }],
+};
+
+describe("<Subtraction />", () => {
+    let history;
+    beforeEach(() => {
+        history = createBrowserHistory();
+        history.push("/subtractions");
+    });
+
+    it("should render", async () => {
```

Comment:
Imo this test is more of a test of `SubtractionsList` than `Subtraction`. I would focus any `Subtraction` tests on really just testing that the page selection/navigation works. Everything else is the called component's job